test/verify/check-pages

```diff
@@ -48,7 +48,7 @@ WantedBy=default.target
 
         # On Atomic no locales other than en_US are supported on the host itself
         # HACK: https://bugzilla.redhat.com/show_bug.cgi?id=1186757
-        elif m.image in [ "continuous-atomic", "fedora-atomic", "rhel-atomic" ]:
+        elif m.atomic_images:
```

Comment:
Typo here, needs to be "image". (I'll wait for the other test results to finish before I re-push).